index.js

```diff
@@ -180,72 +180,91 @@
   //  Value :: Location
   var Value = 'Value';
 
-  //  _funcPath :: (Boolean, Array String, a) -> Nullable Function
-  function _funcPath(allowInheritedProps, path, _x) {
-    var x = _x;
+  //  Prefix :: String
+  var Prefix = 'fantasy-land/';
+
+  //  funcPath :: (Array String, a) -> Nullable Function
+  function funcPath(path, value) {
+    var x = value;
     for (var idx = 0; idx < path.length; idx += 1) {
-      var k = path[idx];
-      if (x == null || !(allowInheritedProps || has(k, x))) return null;
-      x = x[k];
+      x = x && x[path[idx]];
+      if (!x) return null;
```

Comment:
Could we move this to the test in the `for` construct? Also, could we avoid the type coercion?